src/wirecloud/fiware/proxy.py

```diff
@@ -57,23 +61,66 @@ def get_access_token(user, error_msg):
         raise ValidationError(error_msg)
 
 
+def replace_get_parameter(request, gets, token):
+    for get in gets:
+        if get in request['headers']:
+            parameter_name = request['headers'][get]
+            del request['headers'][get]
+
+        url = request['url']
+        if '?' in url:
+            url += "&"
+        else:
+            url += "?"
+
+        url += "{}={}".format(urlquote_plus(parameter_name), urlquote_plus(token))
+        request['url'] = url
+
+
+def replace_header_name(request, headers, token):
+    for header in headers:
+        if header in request['headers']:
+            header_name = request['headers'][header]
+            del request['headers'][header]
+
+            if header_name == "Authorization":
+                token_pattern = "Bearer {token}"
+            else:
+                token_pattern = "{token}"
+
+            request['headers'][header_name] = token_pattern.format(token=token)
+
+
+def replace_body_pattern(request, bodys, token):
+    for body in bodys:
+        if body in request['headers']:
+            pattern = request['headers'][body]
+            del request['headers'][body]
+
+            new_body = request['data'].read().replace(pattern.encode('utf8'), token.encode('utf8'))
+            request['headers']['content-length'] = "{}".format(len(new_body))
+            request['data'] = BytesIO(new_body)
+
+
 class IDMTokenProcessor(object):
+    def __init__(self):
+        self.openstack_manager = OpenstackTokenManager(getattr(settings, 'FIWARE_CLOUD_SERVER', FIWARE_CLOUD_SERVER))
```

Comment:
Here, the constant (the one used as the last parameter) should be `FIWARE_LAB_CLOUD_SERVER`.